service/s3/s3manager/upload.go

```diff
@@ -508,6 +508,7 @@ func (u *multiuploader) upload(firstBuf io.ReadSeeker) (*UploadOutput, error) {
 
 	// Read and queue the rest of the parts
 	for u.geterr() == nil {
+		num++
```

Comment:
I think before my change it sent one part too many
  * send first part outside loop
  * num = 1
  * if num > MaxUploadParts -> no
  * num++ (2)
  * send second part
  * if num > MaxUploadParts -> no
  * num++ (3)
  * send third part
  * if num > MaxUploadParts -> yes, break